@uportal/esco-content-menu/src/components/ContentMenu.vue

```diff
@@ -88,7 +84,7 @@ export default {
     signOutUrl: {type: String, default: process.env.VUE_APP_LOGOUT_URL},
     defaultOrgLogo: {type: String, required: true},
     userInfoPortletUrl: {type: String, default: ''},
-    apiUrlOrgInfo: {type: String, default: ''},
+    switchOrgPortletUrl: {type: String, default: ''},
```

Comment:
Is this used?

Reply:
I'm setting it on my side from hamburger-menu, if we let the property at this place without default value it's to avoid to make a request on a wrong url. In this case it will create on content-user a `<a href="" ...>`.
But what could be the best approach ?